docs/topics/feed-exports.rst

```diff
@@ -458,7 +457,7 @@ For instance::
             'encoding': 'latin1',
             'indent': 8,
         },
-        pathlib.Path('items.csv.gz'): {
+        'items.csv.gz': {
```

Comment:
Why is this changed?

Reply:
sorry. i forgot to revert the changes back.